### PR TITLE
Fix phantomjs installation on Mac

### PR DESCRIPTION
**Pull request by dangoor**
A couple of us just set up new machines and ran into trouble with the installation of PhantomJS. With this change, we only install phantomjs198 which allows the setup to complete successfully.

### CODE REVIEW

File: mac-setup.sh [hunk 1]
```diff
@@ -205,9 +205,8 @@ install_homebrew() {
     brew install apple-gcc42
 
     # We need a particular version of phantomjs to run the end-to-end tests.
+    # This command will let us access those.
     brew tap homebrew/versions
-    brew uninstall --force phantomjs
-    brew install phantomjs198
```
Comment (nsfmc) [inline mac-setup.sh]: oh huh! yeah, we should have moved this into the install_phantomjs function. good catch :+1:

File: mac-setup.sh [hunk 2]
```diff
@@ -298,7 +285,7 @@ install_helpful_tools() {
 
 echo "\n"
 success "Running Khan Installation Script 1.1\n"
-warn "Warning: This is only tested on Mac OS 10.9 (Mavericks)\n"
+warn "Warning: This is only tested on Mac OS 10.9 (Mavericks)\nand Mac OS 10.10 (Yosemite)\n"
```
Comment (nsfmc) [inline mac-setup.sh]: :lollipop: we should really test this from start to finish again. i'll ask karina for the dev laptop

**Comment by nsfmc** [comment]
LGTM

**Comment by dangoor** [comment]
Thanks @nsfmc! Are you going to test this and then land it when it checks out?

**Comment by dangoor** [comment]
Closing this. I've made an update in phabricator.

**Pull request closed without merging**